bitnami/mariadb-galera/templates/headless-svc.yaml

```diff
@@ -23,11 +23,14 @@ spec:
     - name: galera
       port: 4567
       targetPort: galera
+      appProtocol: mysql
```

Comment:
`appProtocol` was introduced in [v1.20](https://kubernetes.io/docs/concepts/services-networking/service/#application-protocol), and we are supporting 1.19+. 
You should check the version in order to add this parameter, you could find an example [here](https://github.com/bitnami/charts/blob/d8a5f63aa65655f819bbd5d31f0be3c6c488e85c/bitnami/mongodb/templates/role.yaml#L23).